synapse/rest/client/receipts.py

```diff
@@ -109,6 +109,46 @@ async def on_POST(
 
         return 200, {}
 
+    async def _is_valid_thread_id(self, event_id: str, thread_id: str) -> bool:
```

Comment:
Maybe call this `_is_event_in_thread` or `_event_in_thread`? We're not doing any validation here, if you see what I mean.

(Sorry for bikeshedding.)